src/lib/trace.c

```diff
@@ -75,7 +75,7 @@ void _trace_event0(uint32_t log_entry)
 	uint32_t message_size_dwords = message_size(0);
 	uint32_t payload_offset = sizeof(struct log_entry_header)
 		/ sizeof(uint32_t);
-	uint32_t dt[message_size_dwords];
+	uint32_t dt[message_size(0)];
```

Comment:
ok, now I see, patch is fine but can you make the message_size macro in upper case (indicates it's pre-calculated at compile time)

Reply:
done